### ISSUE TITLE
Re-work and introduce a "Factions Framework"

### ISSUE THREAD

**Issue by ThatArcanist** [Special-Operations-Brigade/SOB-Issue-Tracker#314]
**Is your feature request related to a problem? Please describe.**
Jaro'tsad has been seeing an increased usage and has become a problem regarding balancing as some units are inconsistently stronger than our typical unit.

**Describe the solution you'd like**
What I propose happening is creating an entire "Base" faction or macros consisting of purely base classes that set the armor values of units. These would be used overwriting any other armor values created by the originating mods. This would allow consistent balancing across all factions in the MTI Factions mod and would also make integration of new units alot easier as the only things that would need to be done is set models, textures and inventory and would remove the need to do any balancing.

**Describe alternatives you've considered**
Balancing each faction individually could be done but takes alot longer to do and will never be consistent with the rest of our factions

**Additional context**
bottom text

**Comment by Rat1452** [comment]
I disagree with having the same armor values all around. Each faction should have its own balance. Jaro shouldn't feel the same as cis. I def think they need to be hit with the balance hammer but it shouldn't be the same for cis. Creating groups that have different levels of toughness gives players a challenge rather than a new face

**Comment by ThatArcanist** [comment]
There would be varying levels of armor, we would create a base light, medium, heavy, etc armor values and what uses them is up to the faction designer at the time but I think we need to have some standardized way of making sure all of the units are balanced around the same target

**Comment by m0kka** [comment]
Yuh, what arcanist said
This will sadly be almost impossible to generalize for vehicles/aircraft, but should be fairly straight-forward for units

**Comment by Hirisu** [comment]
Every faction, except the BNS already have their own base units as to adjust the armor values, wouldnt be hard to unify them in some sense... would just take time:

Edit, turns out i have done it differently for the Jaro's and Revanites so far, by adjusting their uniforms since the units that they are based of are base ArmA indep units

**Comment by Hirisu** [comment]
so is it a yes, do it?
imma get started on it

**Comment by Hirisu** [comment]
Moving this over 

**Jaro'tsad**

- [ ] Wildland Units do not wear the Appropriate uniform, instead reverting back to the base AAF Model (without actually wearing the AAF Uniform)
- [ ] Scav units except Guard have the Same issue as above
- [ ] Vehicles have to be modified to make them feel less unfair
- [ ] HVT Units sometimes spawn with weird/ridicoulus headgear

**Revanites**

- [ ] issues with the headgear being mixed up
- [ ] Armor Values are Very high
- [x] Revanites spawning as indfor instead of opfor?

**CIS**

- [ ] New units dislike using the Actual new paa in their uniforms (haven't worked on it further after doing the first implementation)
- [ ] New units for CIS Human Division as being worked on are: Mortar, Mortar Team member [no mortar weapon, just mortar uniform], EOD, Heavy Machinegunner with the corrosponding Static Machineguns and ParaTroopers

**Common**
- [ ] some units dislike using the Full auto function of the Crappy scav weapons
- [ ] Adding an Actual sniper rifle to the Scav Arsenal to allow for more unit types
- [ ] Making all of the uniforms and Vests that are used by our units into MTI_ equipment to streamline things
